integration-tests/src/test/java/com/datastax/oss/driver/api/core/metadata/NodeStateIT.java

```diff
@@ -81,7 +81,7 @@
               String.format(
                   "load-balancing-policy.filter.class = \"%s$CustomNodeFilter\"",
                   NodeStateIT.class.getName()))
-          .withNodeStateListeners(nodeStateListener)
+          .withNodeStateListener(nodeStateListener)
```

Comment:
The formatter fails for this class, I think because `Session` and `SessionWrapper` are listed as imports that are no longer needed.  Did not know the formatter validated unused imports, cool!